.github/workflows/release.yaml

```diff
@@ -0,0 +1,32 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  actions: write
+  contents: write
+  pull-requests: write
+  statuses: write
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    # don't allow for publishing release unless everything else passes
+    # TODO: not sure if this will work
+    # needs: [Build, Prettier, Unit-Tests, E2E-Tests, Dockerfile-Lint, ESLint, Type-Check]
+    steps:
+      # get the version of the current git tag
+      - name: Get tag version
+        id: vars
+        run: echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+      - name: Generate release
+        uses: 'marvinpinto/action-automatic-releases@v1.2.1'
+        with:
+          repo_token:
+            '${{ secrets.GITHUB_TOKEN }}'
+            # generate release name with the tag name that was just pushed
```

Comment:
This is indented one too far I think

Reply:
This is still wrong

Reply:
Prettier formats this. This is not me...